### ISSUE TITLE
Starter Kit not adding oob/custom webparts to News pages with PnP.Powershell

### ISSUE THREAD

**Issue by pk091990** [pnp/sp-starter-kit#484]
## Category

- [x] Bug

#### Expected or Desired Behavior

Run the deploy kit v2 from [here](https://github.com/pnp/sp-starter-kit) which provisions 3 sites. It should add oob text webparts for all News pages.

#### Observed Behavior
Run the deploy kit as it is without any change. Its not adding any OOB Text webparts in all 4 News Pages given in the xml. Also showing error when I tried to edit the page. Please see the screenshots for the reference. Same issue with custom webparts also.

![image](https://user-images.githubusercontent.com/46436213/112639716-97d41880-8e66-11eb-91ad-261b4fe43d4b.png)

![image](https://user-images.githubusercontent.com/46436213/112639780-a7ebf800-8e66-11eb-9d93-cb45d8da7c6b.png)

#### Steps to Reproduce

Please run the script and open any News Page when script provisioning done.
Webpart is not added.
Show Error while editing the page.

## Environment Details (*Development & Target environment*)
- **Starter kit version**: v2
- **OS**: Windows 10
- **Target Environment**: SharePoint Online
- **Framework**: [Node.js v6/v8/v10/v12]
- **PnPPowerShell Version** : PnP.PowerShell 1.4.0
- **Browser(s)**: Chrome
- **Additional details**: I've checked and its working fine with older version - SharePointPnPPowerShellOnline 3.20.2004.0.

**Comment by bcameron1231** [comment]
Hi! I have reproduced this issue with the latest version of PnP PowerShell. Using a previous version of PnP PowerShell it still works correctly. 

I have a suspicion that we'll need to put in an Issue over at the PnP PowerShell repository https://github.com/pnp/powershell

**Comment by eoverfield** [comment]
@bcameron1231 I wonder if the XML parsing has changed in the way clientside pages canvas controls for text, i.e.

```
<pnp:CanvasControl
  WebPartType="Text"
  ControlId="7819c9ed-fa21-4d71-9b84-71ef785c3dff" Order="2" Column="1">
  <pnp:CanvasControlProperties>
    <pnp:CanvasControlProperty 
      Key="Text"
      Value="&lt;p&gt;&lt;span class=&quot;fontColorThemePrimary&quot;&gt;&lt;span class=&quot;fontSizeMediumPlus&quot;&gt;&lt;strong&gt;BREATHTAKING VIDEOS &amp;amp; PHOTOS&lt;/strong&gt;&lt;/span&gt;&lt;/span&gt;&lt;/p&gt;&lt;p&gt;Contoso MARK8 is the first drone to offer an 8K HDR camera that tilts vertically at and angle of 180&lt;strong&gt;°&lt;/strong&gt;, combined with an up to 2.8X lossless zoom and dolly zoom effect.&amp;nbsp;&lt;/p&gt;&lt;p&gt;&lt;span class=&quot;fontColorBlueDark&quot;&gt;&lt;strong&gt;–&lt;/strong&gt;&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span class=&quot;fontSizeMedium&quot;&gt;&lt;span class=&quot;fontColorBlueDark&quot;&gt;&lt;strong&gt;2.8X FULL HD ZOOM&lt;/strong&gt;&lt;/span&gt;&lt;br&gt;&lt;em&gt;Reveal your inner filmmaker&lt;/em&gt;&lt;/span&gt;&lt;/p&gt;"
    />
  </pnp:CanvasControlProperties>
</pnp:CanvasControl>
```

Let's add to our backlog adding the controls by hand to a page, exporting a template using the latest version of PS, and see if that generates a new template the does work.

**Comment by bcameron1231** [comment]
Yup! Sounds good. I already started doing that and then got distracted. Will try and get a comparison of the news sections later today.

**Comment by pk091990** [comment]
Hi @bcameron1231 , Any update on this ?
